lib/mix/tasks/phx.gen.auth.ex

```diff
@@ -226,6 +226,7 @@ defmodule Mix.Tasks.Phx.Gen.Auth do
       {:eex, "auth_test.exs", Path.join([web_test_prefix, "controllers", web_path, "#{schema.singular}_auth_test.exs"])},
```

Comment:
Users backporting this change to their apps can ignore the changes to this file.